base-system/tools/installer/change-password.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+#	change-password.sh
+#	Script to change the password stored in the core files of a huronOS distribution
+#
+#	Copyright (C) 2023, huronOS Project:
+#		<http://huronos.org>
+#
+#	Licensed under the GNU GPL Version 2
+#		<http://www.gnu.org/licenses/gpl-2.0.html>
+#
+#	Authors:
+#		Daniel Cerna <dcerna@huronos.org>
+
+
+# Tests if a password was supplied
+if [ -z "$1" ]
+  then
+  	echo "Usage: change-password.sh your-new-password"
+  	exit 1
+fi
+
+## Test if the script is started by root user. If not, exit
+if [ "0$UID" -ne 0 ]; then
+	echo "Only root can run $(basename "$0")"; exit 1
+fi
+
+# check for mksquashfs with xz compression
+if [ "$(mksquashfs 2>&1 | grep "Xdict-size")" = "" ]; then
+   echo "mksquashfs not found or doesn't support -comp xz, aborting, no changes made"
+   echo "you may consider installing squashfs-tools package"
+   exit 1
+fi
+
+
+MAIN_PATH="$(pwd)"
+if [ -n "$2" ]; then
+  MAIN_PATH="$2"
```

Comment:
Why not to keep the name `$SYSTEM_MNT`?

Reply:
Because if the script is executed manually by the user, then the context is different and it might not necesarely represent the $SYSTEM_MNT, more likely to be a folder located wherever else in the filesystem